lib/vtt.js

```diff
@@ -1469,4 +1478,4 @@
 
   global.WebVTT = WebVTT;
 
-}(this));
+}(this, (this.vttjs || {})));
```

Comment:
We took a look at using browserify before. If we can separate it from the normal build and not have anything browserify specific in the main `vtt.js` file I think that would be fine.

> Though, now that I think about it, we probably could do this type of API with what we have now, unless you think it's a good idea to be able to have the user pass in the exact shim that should be used.

The less `vtt.js` knows about where `VTTCue` and `VTTRegion` are coming from the better. I'm interested to see with what you come up with!